packages/test-utils/lib/test-utils.js

```diff
@@ -6,6 +6,7 @@ const { Keystone } = require('@keystonejs/keystone');
 const { GraphQLApp } = require('@keystonejs/app-graphql');
 const { KnexAdapter } = require('@keystonejs/adapter-knex');
 const { MongooseAdapter } = require('@keystonejs/adapter-mongoose');
+const { runCustomQuery: runQuery } = require('@keystonejs/server-side-graphql-client');
```

Comment:
Any reason for renaming this here?

Reply:
May be I was tempted to use the one with less words.

Reply:
Yeah, this just adds one more layer of indirection that the user has to manage. I looked at the code itself during review and thought "what on earth is `runQuery`? We don't have a function called `runQuery`!", only to find out that it was just a renaming of `runCustomQuery`.